### ISSUE TITLE
[REQUEST]: Voice Changer Okada Client Closed

### ISSUE THREAD

**Issue by DanteParisii** [w-okada/voice-changer#1411]
### In a few words, describe your idea

Any solution for this error? I tried changing windows many times but the error simply reappears after a few hours or when I restart the PC.

### More information

![275088777-01c8d8cc-1ebd-4ac5-b626-c50ffa5ef205](https://github.com/user-attachments/assets/bbfcda8c-8d87-42e5-9c7a-0144bbb412b2)

**Comment by Kuuko-fokkusugaru** [comment]
The issue lacks information and the screenshot is from a different user report...

**Comment by DanteParisii** [comment]
There is nothing more to explain, the error is that of the image, I open the voice and it just closes, and I use another image because it uninstall the voice, the error is the same for everyone ...

**Comment by Kuuko-fokkusugaru** [comment]
Yeah but you didn't fill your OS, GPU, version of the software, etc... there are many factors that can affect this issue

**Comment by Kuuko-fokkusugaru** [comment]
For example, following the screenshot, I can see that you are executing the software on the desktop, in a folder. You have to run it preferably in a folder inside of the root of one of your drives (better if it is not C if possible).
